parasolr/solr/client.py

```diff
@@ -34,12 +34,17 @@ def as_dict(self):
                 copy[k] = v
         return copy
 
+    def __repr__(self):
+        return 'ParasolrDict(%s)' % super(AttrDict, self).__repr__()
```

Comment:
Probably worth a docstring to document why we needed this (so it doesn't report as `AttrDict`) and maybe why the class needs to be specified in the `super` call (since we want dict repr behavior)